pkg/appconsts/appconsts.go

```diff
@@ -104,7 +104,7 @@ var (
 	// DefaultCodec is the default codec creator used for data erasure
 	// TODO(ismail): for better efficiency and a larger number shares
 	// we should switch to the rsmt2d.LeopardFF16 codec:
```

Comment:
[question] should we remove this TODO because we're switching codecs in this PR? 

```suggestion
```